### PR TITLE
Fix winsock_pointers.h missing in sdist

### PR DESCRIPTION
**Pull request by graingert**
## Scope and purpose

Fixes building from sdist in Windows:
```
iocpsupport.c
twisted_iocpsupport/iocpsupport.c(638): fatal error C1083: Cannot open include file: 'winsock_pointers.h': No such file or directory
error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\VC\\Tools\\MSVC\\14.28.29333\\bin\\HostX86\\x64\\cl.exe' failed with exit status 2
```

## Contributor Checklist:

* [x] The associated ticket in Trac is here: https://twistedmatrix.com/trac/ticket/10103
* [x] I ran `tox -e lint` to format my patch to meet the [Twisted Coding Standard](https://twistedmatrix.com/documents/current/core/development/policy/coding-standard.html)
* [ ] I have created a newsfragment in src/twisted/newsfragments/ (see: [News files](https://twistedmatrix.com/trac/wiki/ReviewProcess#Newsfiles))
* [ ] I have updated the automated tests.
* [x] I have submitted the associated Trac ticket for review by adding the word `review` to the keywords field in Trac, and putting a link to this PR in the comment; it shows up in https://twisted.reviews/ now.

### CODE REVIEW

File: .github/workflows/github-deploy.yml [hunk 1]
```diff
@@ -8,6 +8,24 @@ on:
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: '3.7'
+
+      - name: Install Check-manifest
```
Comment (adiroiban) [inline .github/workflows/github-deploy.yml:22]: looks good... 

Can we have build_wheels as part of `build_sdist` jobs...just after actions/setup-python@v2 

In this way, if check manifest fails, the sdist is not built.

**Review by adiroiban — APPROVED** [review]
Looks good. Thanks!

I think that with twisted-iocpsupport we can consider each PR a  release PR.

So as part of this PR you can also update the version and do the release...so that we don't need another PR for the release.

 But I can help with the review of a separate release PR.

File: .github/workflows/github-deploy.yml (hunk 1 above)
Comment (graingert) [reply .github/workflows/github-deploy.yml:22]: I think it's better if everything runs in parallel

Comment (graingert) [reply .github/workflows/github-deploy.yml:22]: @adiroiban especially as the build is so quick: `Build source distribution (pull_request) Successful in 17s`

**Comment by adiroiban** [comment]
Thanks.

No preview for youtube :)

https://www.youtube.com/watch?v=vCadcBR95oU

**Pull request merged**